### ISSUE TITLE
NTR: gamma-glutamyl cysteine synthetase low in erythrocytes (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7940]
New term request gamma-glutamyl cysteine synthetase low in erythrocytes (segal_060811145857): RBCs, red blood cells

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000132
- label: Reduced erythrocyte gamma-glutamyl cysteine synthetase activity
- parents: [Abnormal erythrocyte enzyme level (HP:0030272)](https://hpo.jax.org/app/browse/term/HP:0030272)
- pmid: PMID:12663448
- def: Activity of gamma-glutamylcysteine synthetase (EC 6.3.2.2) in erythrocytes below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7940
- synonyms: Gamma-glutamyl cysteine synthetase low in erythrocytes(NONE)
